src/components/Loader/index.tsx

```diff
@@ -2,13 +2,12 @@ const Loader = () => {
   return (
     <div className="flex items-center justify-center">
       <svg
```

Comment:
When you change something, you should think about the fact that it changes everywhere, you have now broken the loader that is used in vault and create vault, so either remove it from the page and change it back to 16x16, or add a case for 100x100